lib/galaxy/tools/data/__init__.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, tool_data_path, config_filename=None, tool_data_table_config_
             if not single_config_filename:
                 continue
             self.load_from_config_file(single_config_filename, self.tool_data_path, from_shed_config=False)
+            self.load_length_table(len_file_path, self.tool_data_path, single_config_filename)
```

Comment:
Very minor suggesion. Maybe call this only if `len_file_path` is present? It's not needed otherwise, and you'll be able to get rid of lines 241-242. Seems more straightforward, I think
```if len_file_path:
         self.load_length_table(len_file_path, self.tool_data_path, single_config_filename)
```

Reply:
thank you!